src/quo2/components/navigation/page_nav.cljs

```diff
@@ -0,0 +1,192 @@
+                                [rn/view {:style {:flex-direction :row
+                                                  :align-items    :center}}
+                                 (when horizontal-description?
+                                   [icons/icon mid-section-left-icon
+                                    (icon-props mid-section-main-text-icon-color :big)])
+                                 mid-section-comp-instance]
+                                [rn/view {:style {:flex-direction :row
+                                                  :align-items :center}}
+                                 mid-section-comp-instance
+                                 (when horizontal-description?
+                                   [icons/icon mid-section-left-icon
+                                    (icon-props mid-section-main-text-icon-color :big)])])]
+       :text-with-description mid-section-comp-instance)]))
+
+(defn- right-section-icon
+  [{:keys [background-color icon icon-color push-to-the-left?] :or {push-to-the-left? false}}]
+  [rn/view {:style (assoc
+                    icon-styles
+                    :background-color background-color
+                    :width            32
+                    :height           32
+                    :margin-right (if push-to-the-left? 8 0))}
+   [icons/icon icon (icon-props icon-color :big)]])
+
+(defn page-nav
+  [{:keys [one-icon-align-left? horizontal-description? align-mid? page-nav-color page-nav-background-uri mid-section-type mid-section-icon mid-section-main-text mid-section-left-icon mid-section-right-icon mid-section-description mid-section-description-color mid-section-description-icon mid-section-description-user-icon mid-section-main-text-icon-color left-section-icon left-section-icon-color left-section-icon-background-color right-section-icons]}]
+  (let [{:keys [height width]}      (dimensions/window)
+        put-middle-section-on-left? (or align-mid?
+                                        (> (count right-section-icons) 1))
+        mid-section-props {:mid-section-type                 mid-section-type
+                           :horizontal-description?          horizontal-description?
+                           :mid-section-main-text            mid-section-main-text
+                           :one-icon-align-left?             one-icon-align-left?
+                           :mid-section-right-icon           mid-section-right-icon
+                           :mid-section-icon                 mid-section-icon
+                           :mid-section-main-text-icon-color mid-section-main-text-icon-color
+                           :mid-section-left-icon            mid-section-left-icon}]
+    [rn/view {:style (cond->
+                      {:display            :flex
+                       :flex-direction     :row
+                       :width              width
+                       :height             (* 0.0497 height)
```

Comment:
0.0497 looks suspicious , some kind of magic

Reply:
I'll add a comment indicating where I got the ratio from